java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -159,4 +163,34 @@ public void stopWithStringFailed() {
         sauceSession.stop("failed");
         verify(dummyJSExecutor).executeScript("sauce:job-result=failed");
     }
+
+    @Test
+    public void setsCommandExecutorWithMap(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4813913)